types/log-rotate/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for log-rotate 0.2
+// Project: https://github.com/dstokes/log-rotate#readme
+// Definitions by: sandrewtx08 <https://github.com/sandrewTx08>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function LogRotate(file: string, options: LogRotate.Options, cb: LogRotate.Callback): void;
```

Comment:
```ts
declare function LogRotate(file: string, options: LogRotate.Options, cb: LogRotate.Callback): void;
```
Please consider to change this to 'logRotate' (or 'rotate' as per readme usage). By JS convention function with constructor function bears CamelCase names, not regular, named functions. Thanks! (I understand this would require changing in couple of places).
e.g.:
https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#function_names

Reply:
Hello @peterblazejewicz thanks for the feedback, I applied the convention you mention, could you review it again please?